### ISSUE TITLE
[Bug] Installing never finishes using altstore

### ISSUE THREAD

**Issue by ThePrimeee** [qnblackcat/uYouPlus#1276]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

- iOS/iPadOS version: 17.2.1
- Device model: iPhone 14 Pro Max
- Sideload tool (AltStore, Sideloadly, TrollStore,...): AltStore
- The specific version of uYouPlus (**latest or newest is NOT a version number!**): v18.14.1-3.0

### Describe the bug

When I try to install uYou+ with altstore, the install never finishes, let there for hours. The AppID-s get created but the install never finishes and the app never appears.

### Steps to reproduce the issue

1. Download latest release from github
2. Try to install it with altstore
3. See the bug

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

**Comment by welpdx** [comment]
I got the same issue on ios 17.1.2
What worked for me is restarting the computer. I had installed icloud and itunes but I didnt restart. This was the last thing I did and it worked for me. 

Other things I did:
1. Attach phone by wire to windows pc and unlock phone, accept/trust pc
2. Start AltServer and Alt Store
3. Download Delta and in intunes, I copied the ipa file to delta documents. In Alt Store, I install it from delta documents
4. Login to Alt Store apple ID

Things I did that I dont think affected anything:
1. Added Alt Server folder in windows to AV exclusion
2. Turned off Real Time Protection
3. Allowed App on Firewall
